src/QsCompiler/TestTargets/Libraries/Library1/Library1.csproj

```diff
@@ -6,7 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.11.2006.411-beta" />
+    <PackageReference Include="Microsoft.Quantum.Simulators" Version="0.11.2006.1803-alpha" />
```

Comment:
We probably don't want this version going into the repo, right? This is your local Nuget isn't it?

Reply:
[I responded here.](https://github.com/microsoft/qsharp-runtime/pull/263#discussion_r442381629)